packages/mail-client/src/mail-client-types.ts

```diff
@@ -267,7 +267,7 @@ export interface MailDestination {
  */
 export interface SDKOptions {
   /**
-   * Option to define the strategy of sending emails. The emails will be sent in parallel when setting to true, otherwise in sequential. The default value is true.
+   * Option to define the strategy of sending emails. The emails will be sent in parallel when setting to `true`, otherwise in sequential. The default value is `true`.
```

Comment:
```suggestion
   * Option to define the strategy of sending emails. The emails will be sent in parallel when set to `true`, otherwise in sequential. The default value is `true`.
```